package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "express-rest-api",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
```

Comment:
needs a start script to know how to start this server

Reply:
I installed nodemon , we also had server-static so which one i should use:
`"serve": "static-server src -p 3000"`
or
` "start": "nodemon index.js"`
?

Reply:
nodemon, static-serve is for serving static content, like html and js that loads in browser. This is a backend so not a static resource